### DISCUSSION TITLE
downgrade my organization plan before the next payment date

### DISCUSSION THREAD

**Discussion by songsihyeon** [community/community#49266] [New to GitHub]
I want to downgrade my organization plan before the next payment date.
However, it is inconvenient to use the current plan (team) until the next payment date.
plz help..

**Comment by YANGHONGEUN** [comment]
f482f5411943cacb83aa7ff8d6265abe83ffb0de

**Comment by queenofcorgis** [comment]
👋🏼 @songsihyeon - changes won't go into effect until the end of your billing cycle so the features will be available until your next payment date.
